docs/reference/ccr/use-cases.asciidoc

```diff
@@ -0,0 +1,121 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-use-cases]]
+=== {ccr-cap} use cases
+++++
+<titleabbrev>Use cases</titleabbrev>
+++++
+
+Use {ccr} to enable several use cases within the Elastic Stack:
+
+* <<ccr-disaster-recovery,Disaster recovery>> in case a primary cluster fails,
+with a secondary cluster serving as a hot backup
+* <<ccr-data-locality,Data locality>> to serve reads locally and reduce costly latency
+* <<ccr-centralized-reporting,Centralized reporting>> for preventing search load from interfering with indexing by offloading search to a secondary cluster
+
+Watch the
+https://www.elastic.co/webinars/replicate-elasticsearch-data-with-cross-cluster-replication-ccr[{ccr} webinar] to learn more about the following use cases.
+Then, <<ccr-getting-started,set up {ccr}>> on your local machine and work
+through the demo from the webinar.
+
+[[ccr-disaster-recovery]]
+==== Disaster recovery and high availability
+Disaster recovery provides your mission-critical applications with the
+tolerance to withstand datacenter or region outages. This use case is the
+most common deployment of {ccr}. You can configure clusters in different
+architectures to support disaster recovery and high availability:
+
+* <<ccr-single-datacenter-recovery>>
+* <<ccr-multiple-datacenter-recovery>>
+* <<ccr-chained-replication>>
+* <<ccr-bi-directional-replication>>
+
+[discrete]
+[[ccr-single-datacenter-recovery]]
+===== Single disaster recovery datacenter
+In this configuration, data is replicated from the production datacenter to the
+disaster recovery datacenter. Because the follower indices replicate the leader
+index, {es} can use the disaster recovery datacenter if the production
```

Comment:
It's not really Elasticsearch that can use the indexes replicated to the disaster recovery datacenter, it's that your application can access the replicated indices if the production cluster is unavailable.

Reply:
Changed to read:
>Because the follower indices replicate the leader index, your application can use the disaster recovery datacenter if the production datacenter is unavailable.